source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -491,8 +509,8 @@ export abstract class CardElement extends CardObject {
         Utils.setProperty(result, "separator", this.separator, false);
         Utils.setProperty(result, "height", this.height, "auto");
 
-        if (this.minPixelHeight) {
-            Utils.setProperty(result, "minHeight", this.minPixelHeight + "px");
+        if (this.supportsMinHeight) {
```

Comment:
nit: this is a great variable name- very intuitive to read and understand. thanks!